### ISSUE TITLE
[Core] Sap + Formations + Aggro

### ISSUE THREAD

**Issue by Vishiswaz** [Stormforge-gg/Bugtracker-Fordring#1539]
[//]: # (Enclose links to things related to the bug using http://wowhead.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Zone] A Vision of the Past)
[//]: # ([NPC] Thoralius the Wise)
[//]: # ([Spell][Class] Frostfire Bolt)
[//]: # ([Talent][Class] Brain Freeze)
[//]: # ([Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([Npc][Drop] Thoralius the Wise)
[//]: # ([Web] Armory doesnt work)

**Description**: 
This most often is a problem in instances.

When there's a group of mobs walking in a formation, and I sap one of them, the others continue to walk along their path. This is correct.

However, if I then pull the one sapped mob when the other ones are nowhere nearby, the other ones will automatically aggro as well, and because of chain aggro, everything near them when this happens (and everything between them and me) will also aggro.

Basically, formations should NOT always cause the mobs to aggro when one is pulled. It should be distance-based, like it is with everything else.

**Comment by Vishiswaz** [comment]
Closing due to new server core.
